tests/e2e/pageObjects/notification-page.ts

```diff
@@ -0,0 +1,85 @@
+import { Selector, t } from 'testcafe';
+import { SettingsPage } from './settings-page';
+import { MyRedisDatabasePage } from './my-redis-databases-page';
+
+const settingsPage = new SettingsPage();
+const myRedisDatabasePage = new MyRedisDatabasePage();
+
+export class NotificationPage {
+    //-------------------------------------------------------------------------------------------
+    //DECLARATION OF SELECTORS
+    //*Declare all elements/components of the relevant page.
+    //*Target any element/component via data-id, if possible!
+    //*The following categories are ordered alphabetically (Alerts, Buttons, Checkboxes, etc.).
+    //-------------------------------------------------------------------------------------------
+    // CSS Selectors
+    cssNotificationList = '[data-testid=notifications-list]';
+    cssNotificationTitle = '[data-testid=notification-title]';
+    cssNotificationBody = '[data-testid=notification-body]';
+    cssNotificationDate = '[data-testid=notification-date]';
```

Comment:
Aren't these selectors?

Reply:
Testcafe find() method doesn't work with Selector constructor. For this reason I define css selector and use them in find(). But also I define Title, Body, Date and List as selectors a little bit below like this:
```
    notificationTitle = Selector(this.cssNotificationTitle);
    notificationBody = Selector(this.cssNotificationBody);
    notificationDate = Selector(this.cssNotificationDate);
    notificationList = Selector(this.cssNotificationList);
```
And use selectors where it is needed